### PR TITLE
Add a redirectResolver

### PR DESCRIPTION
**Pull request by garlicnation**
Adds a redirectResolver

### CODE REVIEW

File: src/url-loader/redirect-resolver.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+/**
+ * Resolves a URL using multiple resolvers.
+ */
+export class RedirectResolver implements UrlResolver {
+  constructor(private _config: RedirectConfig) {
+  }
+
+  canResolve(url: string): boolean {
+    return url.indexOf(this._config.redirectFrom) == 0)
```
Comment (rictic) [inline src/url-loader/redirect-resolver.ts]: Can use `url.startsWith(this._config.redirectFrom)`

File: src/url-loader/redirect-resolver.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+import {UrlResolver} from 'polymer-analyzer/lib/url-loader/url-resolver';
+
+export interface RedirectConfig {
+  redirectFrom: string;
+  redirectTo: string
+}
+/**
+ * Resolves a URL using multiple resolvers.
```
Comment (rictic) [inline src/url-loader/redirect-resolver.ts]: This comment is from another class?

File: src/url-loader/redirect-resolver.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {UrlResolver} from './url-resolver';
+
+/**
+ * Resolves a url to another URL on the filesystem.
```
Comment (usergenic) [inline src/url-loader/redirect-resolver.ts]: Dunno mebbe "Resolves a URL having one prefix to another URL a different prefix."

Comment (rictic) [inline src/url-loader/redirect-resolver.ts]: +1

Comment (garlicnation) [inline src/url-loader/redirect-resolver.ts]: Done

File: src/url-loader/redirect-resolver.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  canResolve(url: string): boolean {
+    return url.startsWith(this._redirectFrom);
+  }
+
+  resolve(url: string): string {
+    if (!this.canResolve(url)) {
+      throw new Error(
+          'RedirectResolver cannot resolve: "' + url + '" from: "' +
```
Comment (rictic) [inline src/url-loader/redirect-resolver.ts]: Optional: Use template strings

**Comment by rictic** [comment]
LGTM

File: src/url-loader/redirect-resolver.ts (hunk 4 above)
Comment (garlicnation) [inline src/url-loader/redirect-resolver.ts]: Done

**Pull request merged**